oxidation/libparsec/crates/client_types/tests/test_local_manifest.rs

```diff
@@ -1480,14 +1478,14 @@ fn local_folder_manifest_match_remote() {
     ".png",
 )]
 fn local_folder_manifest_evolve_children_and_mark_updated(
```

Comment:
Maybe you can mark for whole file with `#![allow(clippy::too_many_arguments)]` (and isolate this one ?)

Reply:
@TimeEngineer that's a possibility, but I find this cumbersome (I guess we can enclose the test in a `mod {}` block instead of moving it into another file, but it's still annoying noise for nothing)

I Python I would have modified the linter config to ignore this check in the tests folder, but in Rust it seems complex to do given test and code are intermixed together :/

Reply:
We can also use a struct to reduce the number of arguments

Reply:
I've added the `#[allow]` in the `mod.rs` for the tests